### PR TITLE
Hub #149

### PR DESCRIPTION
**Pull request by balthazzar**
https://github.com/webRunes/Plus-WRIO-App/issues/149

### CODE REVIEW

**Comment by AlexeyAnshakov** [comment]
@michbil please review

File: app.js [hunk 1]
```diff
@@ -1,33 +1,36 @@
 var express = require('express');
-var app = require("./wrio_app.js").init(express);
-var server = require('http').createServer(app).listen(1234);
+var app = require("./wrio_app.js")
```
Comment (michbil) [inline app.js]: @balthazzar This file wan only reformatted, no actual changes made?

File: app.js [hunk 2]
```diff
@@ -1,33 +1,36 @@
...
+	.init(express);
+var server = require('http')
+	.createServer(app)
+	.listen(1234);
```
Comment (michbil) [inline app.js]: @balthazzar i think port should be taken from config.json, 1234 is wrong port

**Comment by michbil** [comment]
Little to late, all seems to be good. One issue only - webrunes session auth wont work with that code, so if you want to make auth working, additional options must be set for express session.

**Comment by AlexeyAnshakov** [comment]
@balthazzar please update the code

**Pull request merged**